### DISCUSSION TITLE
My GitHub copilot can not access my code, what should I do ?

### DISCUSSION THREAD

**Discussion by q2333gh** [community/community#81029] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/github/copilot-docs/assets/32679742/1a9966ac-550a-4c8b-a6d5-a38f7b6fd4e9)
I have grant all permit to copilot .
![image](https://github.com/github/copilot-docs/assets/32679742/417d6e9d-366b-4a9e-b2a8-cbf4011d7ac6)

**Comment by Sandeepkasturi** [comment]
Reinstall the extension, Once again Sign in to Github account, Try once

**Reply by q2333gh** [reply]
I will try it now . thanks for so quick !

**Reply by q2333gh** [reply, marked as answer]
Wahoo. I spend 5 minutes to resintall it . and enable pre-release version of it . It just works! 
Copilot can now read my local codes!
Great! Yeah~
Thank you so much !  @Sandeepkasturi  !
![image](https://github.com/community/community/assets/32679742/6ee1be50-8b07-44dc-a5a6-2e405b2c77bd)

**Comment by q2333gh** [comment]
> Reinstall the extension, Once again Sign in to Github account, Try once
